bower.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "telephone_appointment_planner",
+  "private": true,
+  "dependencies": {
+
```

Comment:
I was in two minds on whether to add the bower stuff at this point when we don't have any dependencies at this moment, but we will eventually have full calendar, list.js etc etc